src/client/components/ShoppingItem/ShoppingItem.js

```diff
@@ -12,8 +12,13 @@ export default function ShoppingItem({
   price,
   initValue,
   isDisable,
+  getCost,
 }) {
-  const [itemValue, setItemValue] = useState(1);
+  const [itemValue, setItemValue] = useState(initValue);
+  React.useEffect(() => {
+    getCost(itemValue * price);
```

Comment:
preferably you can give getSubTotal as you are calculating this and will send it as props

Reply:
get cost is item cost that is related with item